prototyping/dscp-lang-flows/hyproof-token-flows.json

```diff
@@ -0,0 +1,288 @@
+            "metadata_key": "hydrogen_amount",
+            "metadata_value_type": "Literal"
+          }
+        }
+      },
+      {
+        "Op": "And"
+      }
+    ],
+    "version": 1
+  },
+  {
+    "name": "process_issue_cert",
+    "program": [
+      {
+        "Restriction": {
+          "FixedNumberOfInputs": {
+            "num_inputs": 1
+          }
+        }
+      },
+      {
+        "Restriction": {
+          "FixedNumberOfOutputs": {
+            "num_outputs": 1
+          }
+        }
+      },
+      {
+        "Op": "And"
+      },
+      {
+        "Restriction": {
+          "FixedInputMetadataValue": {
+            "index": 0,
+            "metadata_key": "@type",
+            "metadata_value": {
+              "Literal": "InitiatedCert"
+            }
+          }
+        }
+      },
+      {
+        "Op": "And"
+      },
+      {
+        "Restriction": {
+          "FixedInputMetadataValue": {
+            "index": 0,
+            "metadata_key": "@version",
+            "metadata_value": {
+              "Literal": "1"
+            }
+          }
+        }
+      },
+      {
+        "Op": "And"
+      },
+      {
+        "Restriction": {
+          "FixedOutputMetadataValue": {
+            "index": 0,
+            "metadata_key": "@type",
+            "metadata_value": {
+              "Literal": "IssuedCert"
+            }
+          }
+        }
+      },
+      {
+        "Op": "And"
+      },
+      {
+        "Restriction": {
+          "FixedOutputMetadataValue": {
+            "index": 0,
+            "metadata_key": "@version",
+            "metadata_value": {
+              "Literal": "1"
+            }
+          }
+        }
+      },
+      {
+        "Op": "And"
+      },
+      {
+        "Restriction": {
+          "InputHasRole": {
+            "index": 0,
+            "role_key": "hydrogen_owner"
+          }
+        }
+      },
+      {
+        "Op": "And"
+      },
+      {
+        "Restriction": {
+          "FixedInputMetadataValueType": {
+            "index": 0,
+            "metadata_key": "hydrogen_amount",
+            "metadata_value_type": "Literal"
+          }
+        }
+      },
+      {
+        "Op": "And"
+      },
+      {
+        "Restriction": {
+          "OutputHasRole": {
+            "index": 0,
+            "role_key": "hydrogen_owner"
+          }
+        }
+      },
+      {
+        "Op": "And"
+      },
+      {
+        "Restriction": {
+          "OutputHasRole": {
+            "index": 0,
+            "role_key": "energy_owner"
+          }
+        }
+      },
+      {
+        "Op": "And"
+      },
+      {
+        "Restriction": {
+          "FixedOutputMetadataValueType": {
+            "index": 0,
+            "metadata_key": "embodied_co2",
+            "metadata_value_type": "Literal"
+          }
+        }
+      },
+      {
+        "Op": "And"
+      },
+      {
+        "Restriction": {
+          "MatchInputOutputMetadataValue": {
```

Comment:
This is definitely from an old buggy version of `dscp-lang`. You need to use latest `dscp-lang` on `dscp-node/main`

Reply:
What branch / commit hash ?

Reply:
> You need to use latest dscp-lang on dscp-node/main

So `main`